example/ionic-angular/e2e/protractor.conf.js

```diff
@@ -0,0 +1,36 @@
+// @ts-check
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+
+const { SpecReporter, StacktraceOption } = require('jasmine-spec-reporter');
```

Comment:
Also: Unless we're doing some e2e tests (react native project does that), we don't need this file?

Happy to keep it if we're using though.